plugins/pulsar-records-storage/src/main/java/com/github/bsideup/liiklus/pulsar/PulsarRecordsStorage.java

```diff
@@ -175,11 +208,46 @@ public boolean equals(Object o) {
 
         @Override
         public Publisher<Record> getPublisher() {
-            return Flux.using(
-                    () -> {
+            return Flux.usingWhen(createConsumer(), this::consumeMessage, this::cleanConsumer)
+                    .doOnSubscribe(__ -> log.debug(
+                            "pulsar-source: subscription {}, topic {}, partition {} subscribed",
```

Comment:
general logging note: the prefix is pointless, since the logs already include the class (`PulsarRecordsStorage`)

Reply:
removed the prefix